superdesk/io/ingest_provider_model.py

```diff
@@ -139,7 +140,7 @@ def _set_provider_status(self, doc, message=''):
     def on_create(self, docs):
         for doc in docs:
             if doc.get('content_expiry', 0) == 0:
-                doc['content_expiry'] = self.app.config['INGEST_EXPIRY_MINUTES']
+                doc['content_expiry'] = config.INGEST_EXPIRY_MINUTES
```

Comment:
We are getting attribute error. `IngestProviderService` object has no attribute `app`
Will change it to app.config